upgrade/v2.5/README.md

```diff
@@ -75,9 +75,15 @@ Make sure you have all the `IPPools`, `BGPPeers`, `GlobalFelixConfig`, and `Glob
 
 ## 4. Upgrade calico
 
+  > Note: If you are updating a [Canal](https://github.com/projectcalico/canal/tree/master/k8s-install)
```

Comment:
I might also bold the `Note`, e.g. `**Note**`

Reply:
I didn't do that in there because the other note that was already in the file didn't have that.  Should I switch them both?